graph/src/log/elastic.rs

```diff
@@ -272,19 +276,24 @@ impl ElasticDrain {
                         .post(batch_url)
                         .header(CONTENT_TYPE, "application/json"),
                 };
-                header
-                    .body(batch_body)
-                    .send()
-                    .and_then(|response| async { response.error_for_status() })
-                    .map_ok(|_| ())
-                    .unwrap_or_else(move |e| {
+
+                retry("send logs to elasticsearch", &flush_logger)
+                    .limit(max_retries)
+                    .timeout_secs(30)
+                    .run(move || {
+                        header
+                            .try_clone()
```

Comment:
I don't understand why there's the need for the `clone` here

Reply:
This closure may be called multiple times, one for each retry, so it cannot move a captured variable.